articles/active-directory/connect/active-directory-aadconnectsync-feature-directory-extensions.md

```diff
@@ -41,9 +41,21 @@ An object in Azure AD can have up to 100 directory extensions attributes. The ma
 During installation of Azure AD Connect, an application is registered where these attributes are available. You can see this application in the Azure portal.  
 ![Schema Extension App](./media/active-directory-aadconnectsync-feature-directory-extensions/extension3new.png)
 
-These attributes are now available through Graph:  
+These attributes are now available through the **Azure AD Graph**:
+
+We can query the Azure AD Graph through the Azure AD Graph explorer: [https://graphexplorer.azurewebsites.net/](https://graphexplorer.azurewebsites.net/)
 ![Graph](./media/active-directory-aadconnectsync-feature-directory-extensions/extension4.png)
 
+Or through the **Microsoft Graph API**:
+
+We can query the Microsoft Graph API through the Microsoft Graph explorer: [https://developer.microsoft.com/en-us/graph/graph-explorer#](https://developer.microsoft.com/en-us/graph/graph-explorer#)
+
+There are some considerations when working with the Microsoft Graph explorer:
```

Comment:
From this point on, this is probably too much detail since this isn't an article for MS Graph or MS Graph Explorer.  Ideally, there would be a document on how to use graph explorer.  I don't there is one.

Reply:
@tvuylsteke  Thanks for the contribution.  I'd suggest dropping the rest of the detail.

Reply:
I get the point that the article is not on graph. Are you suggesting to drop my addition completely? I still think we need it somewhere and it should be linked here. You can clearly see in the comment section that people are really not finding this info. I've seen customers struggle with this as well. We could drop the info on the AAD Graph and only talk about the Microsoft Graph. The Microsoft Graph is the preferred one I think. @AndKjell Any thoughts from your side?

Reply:
Can we point to https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-graph-api-quickstart#using-the-graph-explorer? I needed, we can add additional information on the Azure AD Graph page.

Reply:
That's a reference to the "old" Azure AD Graph. At the top of that article you have the "we recommend the Microsoft Graph" statement. The Microsoft Graph is documented here: https://developer.microsoft.com/en-us/graph/docs/concepts/overview The docs aren't bad, but you can clearly see that there's a delta with the audience here. "Infra" people get in touch with AAD Connect and all run against the same wall. The doc is way to dev/API oriented.

Still. I get your POV. I guess it's a bad practice documenting how other items work. 

If you agree I'll trim down on the details and just point to this link: https://developer.microsoft.com/en-us/graph/docs/concepts/query_parameters#select-parameter That should get people going as a hint on how to get their property. And I can leave out the clue on how to login/raise permissions as that can be considered basic Microsoft Graph Explorer tasks.